server/test/routes/articleFeeds.test.js

```diff
@@ -0,0 +1,451 @@
+/* eslint-disable */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../../index';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+// object hold users data as user and articles as an array
+const userFollowing = {
+  user: {},
+  articles: []
+};
+const userToFollow1 = {
+  user: {},
+  articles: [],
+};
+const userToFollow2 = {
+  user: {},
+  articles: [],
+};
+const userToFollow3 = {
+  user: {},
+  articles: [],
+};
+const userNotFollowing = {
+  user: {},
+  articles: [],
+};
+
+describe('Testing Users feeds', () => {
+  // users create their accounts
+  before('create account user following others', (done) => {
+    chai.request(server)
+    .post('/api/users')
+    .send({
+      user: {
+        firstName: 'fuserfollowing',
+        lastName: 'luserfollowing',
+        username: 'userfollowing',
+        email: 'userfollowing@gmail.com',
+        password: '123ABab??!'
+      }
+    })
+    .end((err, res) => {
+      Object.assign(userFollowing.user, res.body.user);
+      done();
+    });
+  });
+  before('create account for user to follow1', (done) => {
```

Comment:
Instead of making use of this many before hooks, I suggest you make use of a seed instead

Reply:
Alright